dbt_coves/utils/flags.py

```diff
@@ -101,3 +104,20 @@ def parse_args(self, cli_args: List[str] = list()) -> None:
             if self.task == "check":
                 if self.args.no_fix:
                     self.check["no-fix"] = self.args.no_fix
+
+            if self.args.cls.__name__ == "LoadAirbyteTask":
```

Comment:
Since you changed this check, for consistency purposes you should also update the other `ifs`, like `self.task == "check"`. `self.args.cls.__name__` should be a variable you can reference to stay DRY.

Reply:
Done

Reply:
7f7dcc2[](url)